core/src/test/java/feast/core/model/EntityInfoTest.java

```diff
@@ -66,4 +67,13 @@ public void shouldBuildAndReturnCorrespondingDetail() {
             EntityDetail.newBuilder().setSpec(entitySpec).setLastUpdated(ts).build();
     assertThat(entityInfo.getEntityDetail(), equalTo(expected));
   }
+
+  @Test
+  public void shouldBeEqualToEntityFromSameSpecs() {
+    EntityInfo entity1 = new EntityInfo(entitySpec);
+    entity1.setCreated(Date.from(Instant.ofEpochSecond(1)));
+    EntityInfo entity2 = new EntityInfo(entitySpec);
+    entity1.setCreated(Date.from(Instant.ofEpochSecond(2)));
```

Comment:
Is this line correct? Shouldn't it be `entity2.setCreated(Date.from(Instant.ofEpochSecond(2)));`?